mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -376,3 +378,11 @@ def categories(self) -> List[Optional[str]]:
         assert self.is_categorical
         assert self._values is not None
         return self._values
+
+    @property
+    def meta(self) -> Dict[str, Any]:
+        """
+        Get the tunable's metadata. This is a free-form dictionary that can be used to
+        store any additional information about the tunable (e.g., the unit information).
+        """
+        return self._meta
```

Comment:
What about during the conversion to ConfigSpace?

Reply:
good point! we should add it to the `meta` field there. Let's do it in the next PR, along with the unit tests

Reply:
Please make Issues to track the future PRs so we don't lose track of them.  Thanks!

Reply:
Created issue #419